app/helpers/planning_application_helper.rb

```diff
@@ -19,6 +19,14 @@ def proposed_or_existing(planning_application)
     planning_application.work_status == "proposed" ? "No" : "Yes"
   end
 
+  def map_link(postcode)
+    if postcode.nil?
+      "No location found"
```

Comment:
This would result in `<a href="No location found">Map link</a>` being put on the page, which doesn't seem right. We probably want to just hide the whole `<tr>` if `postcode.blank?`

Reply:
oh yes!!

Reply:
I've removed any logic from the helper method for dealing with an incorrect postcode, because as long as there is some kind of value there, Google will try to create a link